projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/comment-search/comment-search.component.ts

```diff
@@ -27,7 +28,7 @@ export class CommentSearchComponent implements AfterViewInit, OnDestroy {
   }
 
   ngOnDestroy(): void {
-    this.annotationEvents.resetTextHighlight();
+    this.store.dispatch(new fromStore.SearchComment(''));
```

Comment:
does this trigger the same actions that the reset highlight used to do?

Reply:
yes it does

Reply:
yes it does